include/odp/api/spec/crypto.h

```diff
@@ -1,4 +1,5 @@
 /* Copyright (c) 2014-2018, Linaro Limited
+ * Copyright (c) 2021, Nokia
```

Comment:
Update year to 2022

Reply:
Keeping it as 2021 since the code in this PR was published in 2021 and not changed in 2022.